src/libraries/System.Text.Json/gen/JsonSourceGenerator.Parser.cs

```diff
@@ -551,7 +560,7 @@ private static bool TryGetClassDeclarationList(INamedTypeSymbol typeSymbol, [Not
                         INamedTypeSymbol attributeContainingTypeSymbol = attributeSymbol.ContainingType;
                         string fullName = attributeContainingTypeSymbol.ToDisplayString();
 
-                        if (fullName == "System.Text.Json.Serialization.JsonSerializableAttribute")
+                        if (fullName == JsonSerializerAttributeFullName)
```

Comment:
consider renaming to JsonSeializableAttributeFullName to match the attribute name

Reply:
This is changing a string literal to an existing constant of the same value. Changing naming conventions is not in the scope of this PR.

Reply:
I didn't realize you were referring to a typo in the const variable. I guess I can fix that.